pycqed/measurement/waveform_control/segment.py

```diff
@@ -164,6 +182,8 @@ def resolve_segment(self, allow_overlap=False,
         self.resolve_mirror()
         self.resolve_Z_gates()
         self.add_flux_crosstalk_cancellation_channels()
+        if self.resolve_overlapping_elements:
+            self.resolve_overlap()
         self.gen_trigger_el(allow_overlap=allow_overlap)
```

Comment:
Since the trigger elements are generated after resolving overlap, we might create new overlaps in the process.
Since we currently generate all trigger pulses from the master AWG, it is not an immediate issue, but could catch us in the future if we want to trigger from non-master AWGs.
I think a visible comment should be added to the documentation of the `resolve_overlapping_elements` parameters to remind us of this.

Reply:
Added a NB in the documentation of the pulsar parameter.